lambdas/helm-helper-lambda/helmHelperLambda.py

```diff
@@ -40,6 +41,16 @@ def handler(event, context):
             helmupgrade_doc = describe_document(event['ResourceProperties']['HelmUpgradeRunScript'])
             helmelb_doc = describe_document(event['ResourceProperties']['GetElbEndpointRunScript'])
 
+            # First download all helper scripts
+            init_doc = describe_document(event['ResourceProperties']['HelmDownloadScript'])
```

Comment:
I presume this is to cover the scenario where the scripts have been updated?

If so, should it not be inside the Create/Update block i.e. why download the scripts if you're deleting the stack?